pkg/flowaggregator/flowaggregator_test.go

```diff
@@ -32,10 +31,15 @@ import (
 const (
 	testTemplateIDv4        = uint16(256)
 	testTemplateIDv6        = uint16(257)
-	testExportInterval      = 60 * time.Second
+	testActiveTimeout       = 60 * time.Second
+	testInactiveTimeout     = 180 * time.Second
 	testObservationDomainID = 0xabcd
 )
 
+func init() {
+	ipfixregistry.LoadRegistry()
+}
```

Comment:
Do we need this function here? we are using `mockIPFIXRegistry` in testing.

Reply:
removed